### ISSUE TITLE
[Feature Request] Filter Attachments by View

### ISSUE THREAD

**Issue by jpelky** [GorgonUK/Modern-Dropzone-PCF#28]
Perhaps it's already possible, but I am working with a form that uses a large amount of attachments and wanted to use sections of the form to show only specific attachments. I was trying to use a filter on the selected view:
![image](https://github.com/user-attachments/assets/59da06e2-e6c3-45a8-9774-e5f7748de6de)
![image](https://github.com/user-attachments/assets/0f48ee46-0928-4115-852e-1d5cb343d0db)

Being able to filter on Title, or File Type would be really helpful. Thanks for the great work!

**Comment by GorgonUK** [comment]
I see what you mean. Unfortunately, a custom control such as this one is **unable** to inherit the "edit form" specific configurations. The alternative is to either add that functionality to the control itself or to create custom fields for the options menu in the form editor. 

Would this be a viable solution to your proposal?

**Comment by jpelky** [comment]
Oh yeah, absolutely, either option would work great. Thanks for the consideration! I wasn't yet aware custom controls could inherit those configurations, thanks for that.

**Comment by jpelky** [comment]
My current workaround to make these incoming attachments a little easier to locate was adding prefixes to the file names as the Flow attaches them. So, for example, if someone is uploading an ID of some sort, I may prefix it "ID: " which at least helps a little bit

**Comment by jbejai** [comment]
Yes @jpelky , im naming the scans like section 1, section 2 , section 3 etc... but i havent gotten the filter part to work? have you got the filter part to work? saw you had something like "Title begins with" but i'm not seeing that ... wonder if @GorgonUK have a lil workaround for us lol ...:)

**Comment by jbejai** [comment]
Hey,.. guys really need some guidance here plzzz :) ,,, so i add a one column section to a form and inside added a subgrid, for the table for the grid i chose Notes and created a view called section 1. Inside i did the filter thing and got it to work... however when it published it still shows ALL the records :( ... is there any work around for this? i really love this control ,, and there is none like it ... please help if you guys can ... much thanks ... :) ohh it works without adding the control ... but once you add the control it goes away and shows all the documents for that record

![Image](https://github.com/user-attachments/assets/2a447add-2ad2-4196-9ce3-5b633b2080dd)
![Image](https://github.com/user-attachments/assets/cfc88d55-02bb-4b7c-8f85-82924c9600ef)
![Image](https://github.com/user-attachments/assets/d74e3313-e087-45e4-bceb-6d8ed951d246)

**Comment by GorgonUK** [comment]
Hi all, this is being actively developed, will be released later today.

**Comment by jbejai** [comment]
Many many thanks @GorgonUK  ... owe ya one buddy !

**Comment by jbejai** [comment]
I done download this and using it yes... thanks you so much :) :) :)
